plugins/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -212,19 +213,16 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 						name = "�config.language.name�"
 						fileExtensions = "�config.language.fileExtensions�"
 						�IF !config.ecore2Xtext.EPackageInfos.empty�
-							
-							standaloneSetup = {
-								�FOR genmodelURI : config.ecore2Xtext.EPackageInfos.map[genmodelURI.toString].toSet�
-									loadedResource = "�genmodelURI�"
-								�ENDFOR�
-							}
+							�FOR genmodelURI : config.ecore2Xtext.EPackageInfos.map[genmodelURI.toString].toSet�
+								referencedResource = "�genmodelURI�"
+							�ENDFOR�
 						�ENDIF�
 						�IF fromExistingEcoreModels�
 							
 							fragment = ecore2xtext.Ecore2XtextValueConverterServiceFragment2 auto-inject {}
 
-							fragment = adapter.FragmentAdapter {
-								fragment = ecore2xtext.FormatterFragment {}
+							fragment = org.eclipse.xtext.generator.adapter.FragmentAdapter {
```

Comment:
It would be nice to avoid the dependency on the old generator by translating that formatter fragment.